src/main/java/Command/Command.java

```diff
@@ -0,0 +1,8 @@
+package Command;
+
+import TaskList.TaskList;
+
+public abstract class Command {
+    protected static final String SEPARATOR = "_".repeat(30);
```

Comment:
see if you can move some other members from child classes here in addition to the separator.